tests/intensive/sam_video_model_tests.py

```
@@ -0,0 +1,205 @@
+
+    @classmethod
+    def setUpClass(cls):
+        cls.model = foz.load_zoo_model(
+            SAM3_VIDEO_MODEL_NAME,
+            classes=TEXT_PROMPTS,
+            operation_mode="concept",
+        )
+        cls.dataset = _create_video_dataset(num_samples=2, seed=42)
+
+    @classmethod
+    def tearDownClass(cls):
```

Ruff (0.15.15)

[warning] Missing return type annotation for classmethod `tearDownClass`

Add return type annotation: `None`

(ANN206)